.github/workflows/java-versions.yml

```diff
@@ -0,0 +1,36 @@
+name: Java cross-version tests
+
+on:
+  push:
+    branches:
+      - master
+      - "szpak/*"
+      - "marc/*"
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  openjdk14:
+    strategy:
+      matrix:
+        jdk: [8, 11, 14]
+    name: "OpenJDK ${{ matrix.jdk }}"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.jdk }}
+      # Workaround https://github.com/ajoberstar/gradle-stutter/issues/22
+      - name: Reduce number of Gradle regressions builds
+        run: |
+          for l in .stutter/*.lock; do
+            echo $l
+            echo "$(cat $l | grep -v "^#" | tail -1)" > $l
+            cat $l
+          done
```

Comment:
How about just calling one of the compat test tasks or settiong a project property that disables all but one?

Reply:
That would be best and I asked about that once I've got into the project in October:
https://github.com/ajoberstar/gradle-stutter/issues/22

If you know an easy way to tweak it at the Gradle level, we definitely can switch to it.